dynamic-layers/clang-layer/recipes-opencl/igc/intel-graphics-compiler_1.0.8744.bb

```diff
@@ -9,7 +9,7 @@ LIC_FILES_CHKSUM = "file://IGC/BiFModule/Implementation/ExternalLibraries/libclc
                     file://NOTICES.txt;md5=db621145dfb627436bc90ad600386801"
 
 SRC_URI = "git://github.com/intel/intel-graphics-compiler.git;protocol=https;name=igc \
-           git://github.com/intel/vc-intrinsics.git;protocol=https;destsuffix=git/vc-intrinsics;name=vc \
+           git://github.com/intel/vc-intrinsics.git;protocol=https;destsuffix=git/vc-intrinsics;name=vc;nobranch=1 \
```

Comment:
What does this change do?

Reply:
The error I saw was:
```
#WARNING: intel-graphics-compiler-native-1.0.8744-r0 do_fetch: Failed to fetch URL git://github.com/intel/vc-intrinsics.git;protocol=https;destsuffix=git/vc-intrinsics;name=vc, attempting MIRRORS if available
#ERROR: intel-graphics-compiler-native-1.0.8744-r0 do_fetch: Fetcher failure: Unable to find revision e5ad7e02aa4aa21a3cd7b3e5d1f3ec9b95f58872 in branch master even from upstream
#ERROR: intel-graphics-compiler-native-1.0.8744-r0 do_fetch: Fetcher failure for URL: 'git://github.com/intel/vc-intrinsics.git;protocol=https;destsuffix=git/vc-intrinsics;name=vc'. Unable to fetch URL from any source.
#ERROR: Logfile of failure stored in: /project/build-dir/tmp/work/x86_64-linux/intel-graphics-compiler-native/1.0.8744-r0/temp/log.do_fetch.767035
#ERROR: Task (virtual:native:/project/build-dir/../meta-intel/dynamic-layers/clang-layer/recipes-opencl/igc/intel-graphics-compiler_1.0.8744.bb:do_fetch) failed with exit code '1'
```
Actually it is not my fix, I found it in Jeffrey changes when compared it with original code. I guess this change says `bitbake` to ignore the fact that branch doesn't exist for this SHA commit.